### ISSUE TITLE
Ublock doesn't seem to do anything on Firefox immediately after 'reloading the browser'

### ISSUE THREAD

**Issue by mstruijs** [gorhill/uBlock#3625]
### Describe the issue
Ublock doesn't seem to do anything at all on the page visited when opening the browser with the (Firefox) setting 'Show your windows and tabs from last time'. In particular, ads that are otherwise blocked are displayed. A similar issue occurs with the mobile version of Firefox, if the browser has emptied its cache and reloads the 'current' page, Ublock appears ineffective.

Everything seems to work fine after reloading the page once or twice. All other pages appear to be unaffected. Issue appears on FF Nightly and FF Mobile (stable). I think Mobile is supported here, so I think this is sufficient. Otherwise let me know whether to check in a stable FF branch if the problem persists. (I think it does, because stable mobile seems to be having the same problem)

This issue is could be mildly severe, as this means some of the tracking protection uBlock  provides could be disabled as well (not sure whether this is the case, but it is likely)

I have no idea whether this is Firefox messing up or whether the devs of uBlock can solve this, but I think you are best equipped to identify the actual cause.

### One or more specific URLs where the issue occurs

It appears to occur on any valid website whatsoever with ads that would normally be blocked, such as https://youtube.com (Not sure if the main page shows ads. specific videos do show adds otherwise blocked, at least)

### Screenshot in which the issue can be seen

Unnecessary.

### Steps for anyone to reproduce the issue

For the desktop browser:
1. Ensure the setting "Show your windows and tabs from last time" is turned on.
2. Open a web page that has ads that uBlock should block.
3. Close the browser while on that page.
4. Re-open the browser. 
5. The browser should load the page on which you closed the browser.
6. If the bug persists, you should see ads or other things that should be blocked by uBlock.

For the mobile browser:
1. Open a web page that has ads that uBlock should block.
2. Leave the browser app, but do not close it.
3. Somehow force the browser to empty its cache. (I'd guess mobile devs would know how to do this)
4. Enter the browser app. 
5. The browser should (re)load the web page with the ads.
6. If the bug persists, you should see ads or other things that should be blocked by uBlock.

### Your settings

Settings are the same on both devices:
- 'experienced user' is on
- all 'privacy' settings except CSP are on

For desktop:
- OS/version: Windows/ 7 Enterprise SP1 
- Browser/version: Firefox/ 60.0b4 (64-bit) (Aurora channel) (that's what my about page says, anyway)
- uBlock Origin version: v1.15.16

For mobile:
- OS/version: Android/7.0 
- Browser/version: Firefox/ 59.0.1 (no idea if mobile has multiple channels, but mine is probably stable)
- uBlock Origin version: v1.15.16 

##### Your filter lists

Default filter lists + badware + privacy + ... + about 10. 
I think this problem is mostly independent of the lists, as I have mostly different lists on mobile and desktop.
Will expand if people cannot reproduce.

##### Your custom filters (if any)
A few simple cosmetic hiding. Problem persists on pages without custom filters.

**Comment by gwarser** [comment]
https://github.com/gorhill/uBlock/wiki/Advanced-settings#suspendtabsuntilready-experimental

**Comment by mstruijs** [comment]
@gwarser Oh whoops, didn't see that. Well, then this is resolved.

**Comment by mstruijs** [comment]
@gwarser I have a few more comments, though. If you don't want issues for a certain topic, it would be useful to link them from `contributing.md`, to decrease the chance that people make issues for them. I read that document carefully, so I thought my issue was valid. (You might respond that I should have read the 'advanced user' guide when checking the option. I did, but that was a long time ago. If the note was there, I don't remember ).

**Comment by mstruijs** [comment]
@gwarser Second, I've asked for a workaround for this in the browser here https://support.mozilla.org/en-US/questions/1209897 and got a workaround that works for me. Perhaps you can link this thread in you wiki if you think this issue is common enough for a work-around to be nice.

**Comment by WowDude** [comment]
Duplicate of https://github.com/gorhill/uBlock/issues/1327
